### DISCUSSION TITLE
I loved but...

### DISCUSSION THREAD

**Discussion by lok0zzz** [community/community#17332] [Copilot Conversations]
... the copilot is slow and doesn't work well in Python...

oh, and is confusing to use the copilot at the start in the VSCode

**Comment by jfthuong** [comment]
I used it in VS Code for Python and I am amazed everyday by how accurately it can predict.

It works better if you have clear name of functions, variables, type hints, and good docstrings... and also if you write your code in English.

**Reply by rockodagaer** [reply]
Same here!

**Comment by elmarceloc** [comment]
@lok0zzz  "doesn't work well" it's not very clear, It would be nice if you could specify more what you mean
